### PR TITLE
Fix `sphinx` errors and warnings

### PR DESCRIPTION
**Pull request by renatomello**
Checklist:
- [x] Reviewers confirm new code works as expected.
- [x] Tests are passing.
- [x] Coverage does not decrease.
- [x] Documentation is updated.

### CODE REVIEW

File: src/qibo/backends/abstract.py [hunk 1]
```diff
@@ -85,27 +85,37 @@ def compile(self, func):  # pragma: no cover
...
+    def identity_density_matrix(
+        self, nqubits, normalize: bool = True
+    ):  # pragma: no cover
+        """Generate density matrix
+
+        .. math::
+            \\rho = \\frac{1}{2^\\text{nqubits}} \\, \\sum_{k=0}^{2^\\text{nqubits} - 1} \\,
+                |k \\rangle \\langle k|
+
+        if ``normalize=True``. If ``normalize=False``, it return the unnormalized
```
Comment (vodovozovaliza) [inline src/qibo/backends/abstract.py:106]:
```suggestion
        if ``normalize=True``. If ``normalize=False``, returns the unnormalized
```

**Review by vodovozovaliza — APPROVED** [review]

**Review by MatteoRobbiati — COMMENTED** [review]
Thank you for fixing this. 
I confirm errors and warnings are removed while compiling the `html`.
In my case, I still get some warnings, but they seems to be related to the `recommonmark` extension. 
Does this happens to you too?

**Comment by renatomello** [comment]
No, I don't get those

**Comment by MatteoRobbiati** [comment]
> No, I don't get those

Perfect.

**Review by MatteoRobbiati — APPROVED** [review]

**Pull request merged**